_posts/2018-12-06-How-to-do-quick-revie-towards-tickets-and-bugs-before-release.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "How to do quick-review of tickets and bugs before release?"
+date:   2018-12-06 22:49:39 +0800
+
+---
+As we all know, before GA, there are tickets or bugs that we need to decide if they should be fixed or lower their priority or translate to Improvement.
+ This essay is for improving our efficiency toward review ticket.
+
+Before we discuss this topic, I want to share my understanding toward the definition of Bug, P3 bug and improvement.
+
+In my opinion, bug means current flow or operation will cause customer do mistake during work, which means our product not only lowers customer's efficiency, but also leads to customer's trouble.
```

Comment:
'do mistake during work' is very awkward, perhaps change it to 'to have difficulties at work'?